### DISCUSSION TITLE
Unexpected acceleration in step-2 during the shaking table modelling

### DISCUSSION THREAD

**Discussion by CH-max92** [Anura3D/Anura3D_OpenSource#250] [Q&A] [answered]
I am trying to replicate the shaking table test simulation (alsardi&Yerro(2021)).
In the simulation, the shaking table is a rigid body (dry material), and the soil is sandy soil.
Everything seems to be normal during the stress initialization (Step-1) (fig.1). However, after the prescribed velocity (PV) is applied in the step-2, even if the velocity of this time is 0, the model will still have unexpected acceleration and cause the model to collapse (fig.2). In step-2 the mesh seems to move suddenly, causing the horizontal rigid plate to push the soil slope away.
It is speculated that the acceleration may have something to do with the setup of the PV or moving mesh. However, the abnormal acceleration of the second step could not be eliminated after many attempts.

![step1](https://github.com/user-attachments/assets/bd6533e2-71d7-4078-8cc2-bb5fba29303e)
fig.1

![step-2](https://github.com/user-attachments/assets/7ed1a5ce-4dca-4c46-a868-a5fc752c0827)
fig.2

The moving mesh is set as shown in fig.3

![mesh](https://github.com/user-attachments/assets/79bc884b-cb23-415c-a175-88142bdbae97)
fig.3

This problem has troubled me for a long time, and a lot of attempts did not get the right result. Hence, I tried to bring up the discussion in the Anura3D community.
I would like to know how to correct the moving mesh and PV Settings.
Any help or suggestions would be greatly appreciated.
Regards,
Chao.

**Comment by ayerrocolom** [comment, marked as answer]
Hello,

Thank you for your message. 
We need more information to better understand your issue:
What point are you tracking to plot the acceleration? Is this the case with PV=0 m/s? Have you tried linear elastic material for the shaking table? Does this problem also happen if the contact is not activated?

Thanks,

**Reply by CH-max92** [reply]
The PVF file format is incorrect and is now operational. Thank you for your reply.

**Reply by CPLA81192** [reply]
When I use the prescribed velocity and set the moving mesh, when the mesh moves, the material points do not move, which causes the material points to jump out of the mesh. What is the reason for this and how can it be solved?

**Reply by Mangwuha** [reply]
@CH-max92 
Hi，Could you please tell me if it is possible to apply different acceleration files in different directions in Anura3D? For instance, I want to apply A.PVF in the X direction and B.PVF in the Y direction.

**Reply by CH-max92** [reply]
> @CH-max92 Hi，Could you please tell me if it is possible to apply different acceleration files in different directions in Anura3D? For instance, I want to apply A.PVF in the X direction and B.PVF in the Y direction.

I only tested the velocity in horizontal direction. Additionally, in the 2024 version that I tested and used, it seems that only velocity files (PVF) are supported rather than accleration. It is recommended that you conduct searches and tests for the relevant functions in the source code of ANURA3D. From my experience, it is very likely that your expected settings will require secondary development and will be limited by computational efficiency.

**Reply by Mangwuha** [reply]
> > @CH-max92 Hi，Could you please tell me if it is possible to apply different acceleration files in different directions in Anura3D? For instance, I want to apply A.PVF in the X direction and B.PVF in the Y direction.
> 
> I only tested the velocity in horizontal direction. Additionally, in the 2024 version that I tested and used, it seems that only velocity files (PVF) are supported rather than accleration. It is recommended that you conduct searches and tests for the relevant functions in the source code of ANURA3D. From my experience, it is very likely that your expected settings will require secondary development and will be limited by computational efficiency.

Thank you for your reply.
